src/index.js

```diff
@@ -2570,9 +2572,13 @@ const createScatterplot = (initialProperties = {}) => {
         pubSub.publish('pointover', hoveredPoint);
     } else {
       needsRedraw = +hoveredPoint >= 0;
-      if (needsRedraw && !selectionSet.has(hoveredPoint)) {
-        setPointConnectionColorState([hoveredPoint], 0);
-        if (!preventEvent) pubSub.publish('pointout', hoveredPoint);
+      if (needsRedraw) {
+        if (!selectionSet.has(hoveredPoint)) {
+          setPointConnectionColorState([hoveredPoint], 0);
+        }
+      }
+      if (+hoveredPoint >= 0 && !preventEvent) {
```

Comment:
sorry about this. I wanted to make it a clear that we are checking for non-null values as the variable `needsRedraw` is not very telling. Anyway, its your lib so I changed it to your wish :)

Reply:
Thanks for the adjustments! I agree `needsRedraw` is a bad variable name. Any ideas how to improve it? Maybe `previouslyHoveredPoint`?

Reply:
I think its okey since the variable is declared just above it and one can reason what is happening. Otherwise `previouslyHoveredPoint` sounds good or `hasPrevHoveredPoint` to indicate its a boolean

Reply:
I really like `hasPrevHoveredPoint` but I just realized that `needsRedraw` is re-used. So I'll stick to that name for now.